README.md
costaricaraw


~generator/changelog.txt

ChangeLog for PHP XML Sitemap generator

http://www.xml-sitemaps.com
----------------------------------------------

2008-06-15 v3.0

   * gzip content-encoding via http supported


2008-06-15 v2.9

   * automatic email notifications sent every time sitemap is generated
   * html sitemap customization: date format selection (xs_dateformat), order folders by name (xs_htmlnameorder)
   * http/1.1 protocol supported
   * memory usage optimized, new option to define memory limit
   * option to allow pages with http response code different from 200 to be included in sitemap (xs_allow_httpcode)
   * minor bug fixes



2007-12-27 v2.8

   * Live.com ping
   * meta description tag can be (optionally) extracted and inserted in ROR sitemap (advanced settings section)
   * Clear session (or others) parameters from URLs (advanced settings section)
   * IP address connection can be defined (advanced settings section)
   * Progress storage type: "serialize" or "var_export"
   * Changelog order can be changed in config file (xs_chlogorder option)


2007-10-21 v2.7

   * xml sitemap is now template based, which allows to integrate xsl stylesheet into your sitemap. Templates are defined in: sitemap_index_tpl.xml, sitemap_xml_tpl.xml and sitemap_ror_tpl.xml files
   * priority attribute can be assigned automatically by sitemap generator script now, just enable corresponding checkbox in configuration. Priority is descending with depth level: the highest priority is assigned to your homepage and the lower values are set for other pages, depending on how many "clicks" it takes to reach the page
   * minor bug fixes and improvements
   
   
2007-07-30 v2.6

	* sitemap ping for Ask.com and Moreover.com (Live Search)
	* memory usage optimized


2007-05-19 v2.5.3

	* titles are included in ROR sitemap now
	* minor bug fixes and improvements


2007-01-18 v2.5.2

	* minor bug fixes and improvements


2006-11-25 v2.5.1

	* sitemap protocol v0.9 is supported now


2006-11-24 v2.5

	* google sitemap ping URL has been updated
	* an option to automatically remove certain parameters from URLs has been implemented
	* user-agent can be customized now with xs_crawl_ident option in config file
	* robots.txt compliance can be turned on/off now with xs_robotstxt option in config file
	* regular expressions are allowed in "Individual parameters" option - just add a leading '$' sign in the line to enable it


2006-10-11 v2.4

	* yahoo auto notification option added
	* html sitemap is now fully customizable
	* minor bug fixes and improvements


2006-08-18 v2.3

	* ROR sitemaps are now supported
	* you can specify custom port for http connections now using xs_portconnection option in config.inc.php file
	* an option added to limit crawl depth level


2006-07-04 v2.2

	* the <meta name="robots" content="noindex, nofollow"> tag is supported now
	* xml sitemap size (max number of URLs per file) can be specified in config file 
	* location of text sitemap can be specified in config file
	* non-standard ports are supported like example.com:8080
	* minor bug fixes and improvements

2006-03-28 v2.1.1

	* minor bug fixes and improvements

2006-02-21 v2.1

    * html sitemap listing enhancements
	* minor bug fixes

2006-01-17 v2.0

	* HTML Site map generation is now included into the script. You are able to define the number of links per page in html sitemap and customize the page layout using simple template file.
	* Configuration screen interface improved
	* Mutiple minor changes


2005-11-14 v1.2

	* new feature: "Individual attributes" setting at Configuration screen. It allows you to set 
	  specific values for last modification time, frequency andpriority per page. 
	  To use it, define specific frequency and priority attributes in the following format:
	  "url substring,lastupdate YYYY-mm-dd,frequency,priority". 
	* new feature: "Include ONLY URLs" setting. This setting is the opposite to "Exclude URLs". When 
	  it is not empty, ONLY the urls that match the substring entered are included into sitemap.
	* configuration page interface improved (parameters are included into two groups now: main parameters and optional settings)


2005-10-27 v1.1.3

	* the generator creates a sitemap in TEXT FORMAT for you (one URL per line). You can download it from the "Sitemap details" block at the left column of the page. This URLs list can be submitted to Yahoo! at http://submit.search.yahoo.com/free/request


2005-09-14 v1.1.2

	* (fix) asp sites crawling problem


2005-08-30 v1.1.1

	* "sleep" delays are now available to configure to reduce the load on your server
	* new configuration directive: "Do not parse URLs" allows to skip fetching some pages to boost
	  the speed of sitemap generation of the large sites
	* single quoted <base> tag


2005-08-08 v1.1.0

	* "resume the interrupted session" feature added
	* urls extraction has been improved (some uncatched urls are now processed)
	* performance/memory usage optimization


2005-07-31 v1.08

	* generator pages access can be restricted
	* minor improvements


2005-07-20 v1.07

	* CRLF problem with sitemaps (that may lead to a parse problem in rare cases) fixed
	* script configuration auto-filled with initial values	
	* robots.txt is now processed both from top level and from the starting URL directory


2005-07-14 v1.06

	* support of (') character in URLs


2005-07-14 v1.05

	* "Content-type" header is checked and only "text/html" entries are processed
	* "extensions" config settings trailing space problem fixed
	* "gz" added to default excluded extensions
	* "change_frequency" setting fix
	* skip redirections to non-html files 
	* interface enhacements


2005-07-07 v1.04

	* full config with initial settings included


2005-07-07 v1.03

	* "custom lastmod time" bug fixed
	* all pages exluded when "exclude extensions" field is empty


2005-07-05 v1.02

	* robots.txt protocol is supported ('*' and 'googlebot' user-agents are checked)

index.php
<?php

ini_set("session.cookie_lifetime","3600");
header("Expires: Sun 25 Jul 1994 05:00:00 GMT");
header("Last-Modified: " . gmdate("D, d M Y H:i:s") . " GMT");
header("Cache-Control: no-cache, must-revalidate");
header("Pragma: no-cache");
session_start();


// Función para obtener la ip real del usuario
// Verifica si está en un proxy, ip compartido, etc
$idaff = $_REQUEST['idadffi'];
$idbanner = $_REQUEST['idbanner'];
$Url = $_REQUEST['url'];
include("includes/configuracion.php");

$db = new MySQL();


if($idaff != "")
{
    $resp = $db->consulta("SELECT * FROM `affiliate_banner_counter` WHERE `idaffiliates` = '".$idaff."' and `idbanner` = '".$idbanner."'" );
    $Page = $db->fetch_array($resp);

    $resp1 = $db->consulta("SELECT `link` FROM `banner` WHERE `id` = '".$idbanner."'" );
    $Page1 = $db->fetch_array($resp1);
    if(!empty($Page))
    {
        $update = $db->consulta( "UPDATE `affiliate_banner_counter` SET `count` = (`count` + 1) WHERE `idaffiliates` = '".$idaff."' and `idbanner` = '".$idbanner."'");
    }
    else
    {
        $update = $db->consulta( "insert into `affiliate_banner_counter` (`idaffiliates`,`idbanner`,`count`) values ('".$idaff."', '".$idbanner."', 1)");

    }
    ?>
    <script> location.replace("<?php echo $Page1['link']?>"); </script>
<?php
}



//if($_SESSION["time"]){
include('includes/sesiones.php');
//}
$language = new Language();
$lang = $language->getLanguage(@$_POST['lang']);
$Idioma = $_SESSION['LANGUAGE'];
$cmd = $_REQUEST['cmd'];
$carro=$_SESSION['carro']; 
$permalinks = explode("/",$_SERVER['REQUEST_URI']);
if($_REQUEST['kid']){
$_SESSION['kid']=$_REQUEST['kid'];
$ip   = $_SERVER["REMOTE_ADDR"];
$refiere = $_SERVER['HTTP_REFERER'];
$fecha_hora = gmdate("Y-m-d, g:i:s", time() + $zone);
$navegador = $_SERVER["HTTP_USER_AGENT"];
$kid = $_REQUEST['kid'];
$CuentaAfiliado = $db->consulta("SELECT * FROM `affiliates` WHERE `id`='".$kid."'");
$Visitas=$db->fetch_array($CuentaAfiliado);
$Contador = $Visitas['counter'] + 1;
$ActualizarAfiliado = $db->consulta("UPDATE `affiliates` SET `counter`='".$Contador."' WHERE `id`='".$Visitas['id']."'");
$Log = $db->consulta("INSERT INTO `affiliates_log` (`ip`,`link`,`browser`,`dateTime`,`affiliates_id`) VALUES ('".$ip."','".$refiere."','".$navegador."','".$fecha_hora."','".$Visitas['id']."')");	
}
//Este es el index.php donde se manerajá el Request
$permalinks = explode("/",$_SERVER['REQUEST_URI']);
if($permalinks[1]=="home"){
$ip   = $_SERVER["REMOTE_ADDR"];
$refiere = $_SERVER['HTTP_REFERER'];
$fecha_hora = gmdate("Y-m-d, g:i:s", time() + $zone);
$navegador = $_SERVER["HTTP_USER_AGENT"];
$Afiliado = $_REQUEST['afid'];
$CuentaAfiliado = $db->consulta("SELECT * FROM `affiliates` WHERE `contact`='".$permalinks[2]."'");
$Visitas=$db->fetch_array($CuentaAfiliado);
$Contador = $Visitas['counter'] + 1;
$ActualizarAfiliado = $db->consulta("UPDATE `affiliates` SET `counter`='".$Contador."' WHERE `id`='".$Visitas['id']."'");
$Log = $db->consulta("INSERT INTO `affiliates_log` (`ip`,`link`,`browser`,`dateTime`,`affiliates_id`) VALUES ('".$ip."','".$refiere."','".$navegador."','".$fecha_hora."','".$Visitas['id']."')");
$_SESSION['affiliateID'] = $Visitas['id'];
$_SESSION['affiliateName'] = $Visitas['name'];
$_SESSION['topAdventure'] = $Visitas['top'];
$_SESSION['time'] = time();
$_SESSION['affiliateHome'] = $Visitas['url'];	
$_SESSION['affiliateLogo'] = $Visitas['logo'];
}
?>
<!DOCTYPE html>
<html lang="en"><!-- InstanceBegin template="/Templates/crraw.dwt.php" codeOutsideHTMLIsLocked="false" -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="google-translate-customization" content="a519e97f3465566b-05182cbb86aa22ce-g1b10ff64d7660cc9-14"></meta>

<?php include('includes/metas.php'); ?>
<!-- InstanceBeginEditable name="doctitle" -->

<meta name="viewport" content="width=device-width, height=device-height" />
<meta name="viewport" content="user-scalable=yes" />
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="apple-mobile-web-app-capable" content="no"> 
<meta name="apple-mobile-web-app-status-bar-style" content="black">
<title>Costa Rica Raw Adventures</title>
<link rel="icon" href="favicon.png" type="image/png" />
<link rel="bookmark" href="favicon.ico" />
<link rel="shortcut icon" href="favicon.ico" />

<meta property="og:title" content="Costa Rica Raw Adventures" />
<meta property="og:type" content="website" />
<meta property="og:image" content="https://www.costaricaraw.com/dev/images/facebook_share_100.jpg" />
<meta property="og:url" content="https://www.costaricaraw.com/dev/" />
<meta property="og:description" content="Costa Rica Raw | Our trekking adventures are done with the bare essentials, just as the natives would have.. Two feet on the ground, and sometimes your hands & more in the dirt! Raw Adventures are designed for you to challenge yourself and allows you to choose when you have reached your limit. " />

     
 <?php

$puerto = $_SERVER['SERVER_PORT'];
if($puerto=="443"){
?>
<base href="https://www.costaricaraw.com/dev" />
<?php
} else {
?>
<base href="http://www.costaricaraw.com/dev" />
<?
}
?> 
<!-- InstanceEndEditable -->
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->

<link href="css/style.css" rel="stylesheet" type="text/css" />

<!--    livechat-->
<!--    <script type='text/javascript'>function chatwoo_a() {var s = document.createElement('script');s.type = 'text/javascript';s.src = "https://chatwoo.com/c1.jsp?host=" + window.location.host + "&hostname=https://chatwoo.com";document.getElementsByTagName('head')[0].appendChild(s);}function chatwoo_d(r) {var s = document.createElement('script');s.type = 'text/javascript';s.src = r.d;document.getElementsByTagName('head')[0].appendChild(s);}chatwoo_a();</script> <noscript>Powered by<a href='https://chatwoo.com' title='Free Live Chat for website'>chatwoo.com</a></noscript>-->
</head>

<body>
<div id="Principal">

<div class="mainWrap">
<div class="leftWrap">

  <table width="880" border="0" align="center" cellpadding="0" cellspacing="0" >
    <tr>
    <td>
  
      <table width="880" border="0" cellspacing="0" cellpadding="0">
        <tr>
          <td height="150" style="background-image: url('images/crra-header.jpg'); background-repeat: no-repeat; ">
            <div id="Content" style="position: absolute;">
              <div id="Compas"><div id="MapaLink" onClick="location.href='index.php?cmd=crmap'"></div></div>
              <div id="Slogan"><img src="images/slogan.png" border="0" alt="THE PATHS-PRESERVING THE FUTURE" title="THE PATHS-PRESERVING THE FUTURE" /></div>
              </div>
          </td>
          <td width="30">&nbsp;</td>
          </tr>
        <tr>
          <td>&nbsp;</td>
          <td width="10">&nbsp;</td>
          </tr>
        <tr>
          <td><table width="100%" border="0" cellspacing="0" cellpadding="0">
            <tr>
              <td width="86%" height="50" align="right" valign="bottom" style="background:url(images/bgTop.png) no-repeat bottom left;"><form name="language" id="language" method="post">
                <input type="hidden" name="lang" id="lang" />
                <table border="0" cellspacing="2" cellpadding="2" style="margin-right:45px;">
                  <tr>
                      <!--carrito de compra-->
                      <td align="center">
                          <a href="https://www.costaricaraw.com/dev/index.php?cmd=cart" title="View Shopping Cart"><strong>Shopping Cart</strong>
                              <img src="images/cart.png" alt="View Shopping Cart" width="24" height="24" border="0" align="absmiddle" title="View Shopping Cart" />
                              <?php
                              if($carro){
                                  $contador=0;
                                  $suma=0;

                                  foreach($carro as $k => $v){
                                      $subto=$v['cantidad']*$v['precio'];
                                      $suma=$suma+$subto;
                                      $cant = $v['cantidad'];
                                      $cantidad = $cantidad + $cant;

                                      $contador++;
                                  }

                                  echo "Items: ". $cantidad;
                              }else{
                                  echo "Items: 0";
                              }
                              ?>
                          </a>
                      </td>
                      <!--/carrito de compra-->
                    <td width="10">&nbsp;</td>
                    <td><!-- InstanceBeginEditable name="Traductor" -->&nbsp;
        <!-- InstanceEndEditable --></td>
                    </tr>
                  </table>
                </form></td>
              </tr>
            <tr >
              <td height="400" style="background:url(images/bgBody.png) repeat-y left; height:100%">
                <table width="99%" border="0" align="right" cellpadding="0" cellspacing="0">
                  <tr>
                    <td width="505"><table width="98%" border="0" cellpadding="0" cellspacing="0" style="margin-left:10px;">
                      <tr>
                        <td width="198" valign="top" class="MenuDiv"><div id="MenuDiv">
                          <table width="100%" border="0" cellspacing="0" cellpadding="0">
                            <tr>
                              <td>&nbsp;</td>
                              </tr>
                            
                            <tr>
                              <td>&nbsp;</td>
                              </tr>
                            <tr>
                              <td><?php 
								include(_HTML_MENU);
                              ?></td>
                              </tr>
                            <tr>
                              <td>&nbsp;</td>
                              </tr>
                            <tr>
                              <td>&nbsp;</td>
                              </tr>
                            </table>
                            
                          
                          <?php
$puerto = $_SERVER['SERVER_PORT'];
if($puerto=="443"){ 

} else {
?>
                          </div>
                          <br />
                          <!--div><strong><?php echo Translate($Idioma,"Weather");?></strong></div>
                          <iframe src="modulos/weather/index.php" width="150" height="100" scrolling="no" frameborder="0"></iframe-->
                          <?
} 
?>
                          
                          </td>
                        <td width="550" valign="top">
                          <table width="98%" border="0" cellpadding="0" cellspacing="0">
                            <tr><td><!-- InstanceBeginEditable name="WebContent" -->
   
<?php include('modulos/modulos.php'); ?>
​
<!-- InstanceEndEditable --></td>
                              </tr>
                            </table>
                          </td>
                        </tr>
                      </table></td>
                    </tr>
                  </table>              </td>
              </tr>
            <tr >
              <td width="850" height="72" valign="top" style="background:url(images/bgFooter.png) no-repeat left top;"><table width="92%" border="0" align="center" cellpadding="0" cellspacing="0">
                <tr>
                  <td>&nbsp;</td>
                  
                  </tr>
                <tr>
                  <td align="center"><?php include(_HTML_FOOTER); ?></td>
                  </tr>
                </table></td>
              </tr>
          </table></td>
          <td width="10" valign="top">&nbsp;</td>
          </tr>
        <tr>
          <td>&nbsp;</td>
          <td width="10">&nbsp;</td>
          </tr>
        <tr>
          <td>&nbsp;</td>
          <td width="10">&nbsp;</td>
          </tr>
        <tr>
          <td>&nbsp;</td>
          <td width="10">&nbsp;</td>
          </tr>
    </table></td></tr>
  </table>
   </div>
	
	<div class="rightWrap"><div class="smartBannerIdentifier"></div>
		<div class="banner">
		
			<?php include('modulos/banner.php'); ?>
		
		</div>
	</div>

</div>
  
  
</div>
<!--<script type="text/javascript" src="lib/jquery.js"></script>-->
<script src="js/jquery.min.js" type="text/javascript"></script>
<script src="js/jquery.bpopup.min.js" type="text/javascript"></script>

<?php
if($_REQUEST["cmd"]!="book"){
    ?>
    <!--Start of Tawk.to Script
    <script type="text/javascript">
        var Tawk_API=Tawk_API||{}, Tawk_LoadStart=new Date();
        (function(){
            var s1=document.createElement("script"),s0=document.getElementsByTagName("script")[0];
            s1.async=true;
            s1.src='https://embed.tawk.to/5796127592a0df3b7f408c8f/default';
            s1.charset='UTF-8';
            s1.setAttribute('crossorigin','*');
            s0.parentNode.insertBefore(s1,s0);
        })();
    </script>
    End of Tawk.to Script-->
<?php
}
?>



</body>

<!-- Start of Chatwoo -->
<!--<script type='text/javascript'>
    function chatwoo_a() {
        var s = document.createElement('script');
        s.type = 'text/javascript';
        s.src = 'https://chatwoo.com/c1.jsp?host=' + window.location.host + '&hostname=https://chatwoo.com/' ;
        document.getElementsByTagName('head')[0].appendChild(s);
    }
    function chatwoo_d(r) {
        var s = document.createElement('script');
        s.type = 'text/javascript';
        s.src = r.d;
        document.getElementsByTagName('head')[0].appendChild(s);
    }
    chatwoo_a();

</script>
<noscript>Powered by <a href='https://chatwoo.com' title='Free Live Chat for website'>chatwoo.com</a></noscript>-->
<!-- End of Chatwoo-->

<!-- InstanceEnd --></html>
